cmd/migrate_loadtest/migrate_loadtest.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/block-explorer/blob/master/LICENSE.md.
+
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/insolar/insconfig"
+	"gopkg.in/gormigrate.v1"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+
+	"github.com/insolar/block-explorer/configuration"
+	"github.com/insolar/block-explorer/instrumentation/belogger"
+	"github.com/insolar/block-explorer/migrations"
+)
+
+func main() {
```

Comment:
What about adding flag to existed `migrate` instead of new cmd?

Reply:
done